src/interpreter/kernel/exec.js

```diff
@@ -991,6 +991,37 @@ export function executeStackFrame(frame: StackFrame, depth: number = 0): any {
         throw new RuntimeError('Unsupported operation ' + instruction.id + ' on ' + instruction.object);
       }
 
+      break;
```

Comment:
Btw do you know if there a rule to ensure the break in ESLint? Sometimes I forget it and loose a lot of time trying to debug it.

Reply:
I don't think so https://github.com/eslint/eslint/issues/7697